go.mod

```diff
@@ -67,6 +67,7 @@ require (
 	golang.org/x/sync v0.0.0-20201020160332-67f06af15bc9
 	golang.org/x/sys v0.0.0-20210315160823-c6e025ad8005
 	gopkg.in/check.v1 v1.0.0-20200227125254-8fa46927fb4f // indirect
+	gopkg.in/yaml.v2 v2.4.0
```

Comment:
Can we use `yaml.v3` below?  I'd like to avoid vendoring multiple versions of the same library.

Reply:
It should work with `yaml.v3`. I have just changed the code.